pages.zh/common/gpg2.md

```diff
@@ -0,0 +1,49 @@
+# gpg2
+
+> GNU Privacy Guard 2.
+> GNU Privacy Guard 1 请参见 `gpg`。
+> 更多信息: <https://docs.releng.linuxfoundation.org/en/latest/gpg.html>.
+
+- 生成密钥：
+
+`gpg2 --gen-key` 或者 `gpg2 --full-gen-key`
```

Comment:
This example code itself should match the following regular expression:
```
^[^`]+`$
```
which this does not&mdash;Chinese letters interrupt this example.  Please correct this :)